modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/LocalPartitionReplicaEventParameters.java

```diff
@@ -27,13 +27,18 @@ public class LocalPartitionReplicaEventParameters implements EventParameters {
     /** Zone partition id. */
     private final ZonePartitionId zonePartitionId;
 
+    /** Event's revision. */
+    private final long revision;
```

Comment:
We have a special CausalEventParameters for this case

Reply:
Done